sdk/servicebus/service-bus/src/util/errors.ts

```diff
@@ -66,15 +66,8 @@ export function getClientClosedErrorMsg(entityPath: string): string {
  * @internal
  * Gets the error message when a sender is used when its already closed
  * @param entityPath Value of the `entityPath` property on the client which denotes its name
- * @param isClientClosed Denotes if the close() was called on the client that created the sender
  */
-export function getSenderClosedErrorMsg(entityPath: string, isClientClosed: boolean): string {
-  if (isClientClosed) {
-    return (
-      `The client for "${entityPath}" has been closed. The sender created by it can no longer be used. ` +
-      `Please create a new client using an instance of ServiceBusClient.`
-    );
-  }
+export function getSenderClosedErrorMsg(entityPath: string): string {
```

Comment:
The error message `Please create a new client using an instance of ServiceBusClient.` is not applicable since we don't have any QueueClient or TopicClient, and we can only be throwing `connection closed` or the `sender closed` errors and that makes the `isClientClosed` check irrelevant. Even if the client is closed, we should only be throwing the error to create a new sender using the createSender from SBC.

Reply:
I should do the same to receiver error message, but the TODOs in tests did not require me to touch that method.
Let me know if you have any concerns with the approach, I'll continue the same for receiver once you wave the green flag.

Reply:
Oh, makes total sense. I'm good if you want to wait until next PR.

Reply:
Will do a followup PR.